Gemfile.lock

```diff
@@ -315,76 +316,78 @@ GEM
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
```

Comment:
### [concurrent-ruby](http://www.concurrent-ruby.com)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.2.2 → 1.2.3

[_[change-log](https://github.com/ruby-concurrency/concurrent-ruby/blob/master/CHANGELOG.md), [source-code](https://github.com/ruby-concurrency/concurrent-ruby), [gem-diff](https://my.diffend.io/gems/concurrent-ruby/1.2.2/1.2.3)_]





<details>
<summary>Commits</summary>

A change of **25** commits. See the full changes on [the compare page](https://github.com/ruby-concurrency/concurrent-ruby/compare/v1.2.2...v1.2.3).

These are the first 10 commits:
- (7d58ec8) [Check the first Concurrent::Map default_proc argument is a C…](https://github.com/ruby-concurrency/concurrent-ruby/commit/7d58ec8998c226fd407fc7632b40bebdb631b8ca)
- (c587821) [Update comment about ::Hash on CRuby, it's not fully thread-…](https://github.com/ruby-concurrency/concurrent-ruby/commit/c587821ba5dd7e3fa0f758e0624ee4e47423f8f1)
- (e7ff945) [Update comment about ::Array on CRuby, it's not fully thread…](https://github.com/ruby-concurrency/concurrent-ruby/commit/e7ff945661a93a2e094b3259f23ccb1b3eea9bcf)
- (9f40827) [Fix TimerTask `:execution_interval` docs](https://github.com/ruby-concurrency/concurrent-ruby/commit/9f40827be9a8a192a6993a8d157bd3ed0662ada0)
- (483dd33) [Fix TimerTask docs to not refer to `#execute` as "blocking" …](https://github.com/ruby-concurrency/concurrent-ruby/commit/483dd33720f57799a913c9ae4419930630a39a1f)
- (3bfe39e) [Fix TimerTask example output (#1003)](https://github.com/ruby-concurrency/concurrent-ruby/commit/3bfe39e77958876403cefa3a2d89e86c7bdb5f9d)
- (bf966c0) [Fix broken CI due to rake-compiler error on Ruby < 2.6](https://github.com/ruby-concurrency/concurrent-ruby/commit/bf966c05654c490ec8f4db8e5ab2bf9b7e64a006)
- (1982b92) [Fix doc typo: yeild → yield](https://github.com/ruby-concurrency/concurrent-ruby/commit/1982b92daa8aee0d88db5212a61b790142c4106f)
- (42d2766) [Fix DaemonThreadFactory - reuse single Java thread factory](https://github.com/ruby-concurrency/concurrent-ruby/commit/42d2766023ffe5e95b62b46d29fe9a796af6a3fe)
- (dadc2ad) [Fix sporadic failures testing with JRuby (#1012)](https://github.com/ruby-concurrency/concurrent-ruby/commit/dadc2ad81f8c57a03400585187c3405c96713198)


</details>